delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/MigrationState.scala

```diff
@@ -5,4 +5,6 @@ object MigrationState {
 
   def isRunning: Boolean = sys.env.isDefinedAt("MIGRATE_DATA")
 
+  def isIndexingDisabled: Boolean = sys.env.isDefinedAt("DISABLE_INDEXING")
```

Comment:
why is this an env variable and not part of the `migration` configuration?

Reply:
Because this is something temporary and less intrusive that passing on the migration config in other modules